src/screens/SettingsModal.js

```diff
@@ -169,91 +162,87 @@ export default function SettingsModal() {
 
   const memoSettingsOptions = useMemo(() => settingsOptions(colors), [colors]);
   return (
-    <Modal
-      minHeight={height - 100}
-      onCloseModal={goBack}
-      radius={18}
-      showDoneButton={ios}
-      skipStatusBar={android}
-      testID="settings-modal"
+    <Box
+      background="body"
+      flexGrow={1}
+      {...(android && { borderTopRadius: 30 })}
     >
-      <Container>
-        <Stack.Navigator
-          screenOptions={{
-            ...memoSettingsOptions,
-            headerRight: renderHeaderRight,
+      <Stack.Navigator
+        screenOptions={{
+          ...memoSettingsOptions,
+          headerRight: renderHeaderRight,
+        }}
+      >
+        <Stack.Screen
+          name="SettingsSection"
+          options={{
+            cardStyleInterpolator,
+            title: lang.t('settings.label'),
           }}
         >
-          <Stack.Screen
-            name="SettingsSection"
-            options={{
-              title: lang.t('settings.label'),
-            }}
-          >
-            {() => (
-              <SettingsSection
-                onCloseModal={goBack}
-                onPressBackup={onPressSection(SettingsPages.backup)}
-                onPressCurrency={onPressSection(SettingsPages.currency)}
-                onPressDev={onPressSection(SettingsPages.dev)}
-                onPressLanguage={onPressSection(SettingsPages.language)}
-                onPressNetwork={onPressSection(SettingsPages.network)}
-                onPressPrivacy={onPressSection(SettingsPages.privacy)}
-              />
-            )}
-          </Stack.Screen>
-          {Object.values(SettingsPages).map(
-            ({ component, getTitle, key }) =>
-              component && (
-                <Stack.Screen
-                  component={component}
-                  key={key}
-                  name={key}
-                  options={{
-                    cardStyleInterpolator,
-                    title: getTitle(),
-                  }}
-                  title={getTitle()}
-                />
-              )
+          {() => (
```

Comment:
why is this a function?

Reply:
Oh this was already here. The only diff in this file is that I have removed the top level <Modal> component.